electricpy/__init__.py

```diff
@@ -4695,47 +4749,57 @@ def characterz(R, G, L, C, freq=60):
     return (Zc)
 
 #define propagation_constants for long transmission line
-def propagation_constants(z, y, length):
+def propagation_constants(z, y, Length):
     
-    '''
-        this function calculates the propagation constants of long transmission line 
-        
-        long transmission line is governed by differential equation
-        
-        (d^2)V/d(x^2) = (gama)*V
-        
-        Parameters
-        ----------
-        z: impedence of the transmission line R+j*2*pi*f*L
-        
-        y: admitance of the transmission line g+j*2*pi*f*C
-        
-        Returns
-        -------
-        
-        gama: propagation constant
-        
-        zc: surge_impedence
-        
-        alpha : attenuation constant
-        
-        beta: imaginary part of gama
-        
+    """
+    Transaction Line Propagation Constant Calculator
+
+    This function will evaluate the propagation constants for a long transmission
+    line whose properties are governed by the differential equation:
     
-    '''
+    .. math:: \\frac{d^2V}{dx^2} = \\gamma V
+    
+    From the above equation, the following formulas are derived to evaluate the
+    desired constants.
+    
+    .. math:: \\gamma = \\sqrt( z * y )
+    
+    .. math:: Z_{\\text{surge}} = \\sqrt( z / y )
+    
+    .. math:: \\alpha = \\Re{ \\gamma }
+    
+    .. math:: \\beta = \\Im{ \\gamma }
+    
+    Parameters
+    ----------
+    z:              complex
+                    Impedence of the transmission line: R+j*2*pi*f*L
+    y:              complex
+                    Admitance of the transmission line g+j*2*pi*f*C
+        
+    Returns
+    -------
+    params:    dict
+               Dictionary of propagation constants including:
+                    
+                         gamma:   Propagation constant
+                         zc:            Surge impedance
+                         alpha:      Attenuation constant
+                         beta:        Imaginary portion of gamma
+    """
 
-    assert length > 500, "long transmission line length should be grater than 500km"
+    assert Length > 500, "long transmission line length should be grater than 500km"
```

Comment:
I'm sorry, I probably didn't make my previous request clear. the variable `length` was find being lower-case, I was hoping we could make the word "long" capitolized as: "Long" so that the assert statement reads: "Long transmission line...."

Reply:
completed in 9e81883ef1b4b12f6e1d23e2badc681666b68951